src/index.html

```diff
@@ -11,7 +11,145 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      method="post"
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application">
+      <fieldset class="fieldset__margin">
+        <legend>Personal information</legend>
```

Comment:
WDYT, writing tag and text in one line it's that a good idea?

```suggestion
        <legend>
          Personal information
        </legend>
```